### PR TITLE
Move Acknowledgements note up to fix duplicate sections

### PR DESCRIPTION
**Pull request by AnilSeervi**

### CODE REVIEW

**Comment by AnilSeervi** [comment]
It still creates duplicate headlines.🤔

**Comment by rebloor** [comment]
@AnilSeervi The duplication comes from a "manually" inserted Example extensions section in lines 39 to 43. Delete these lines and move the browser compatibility/extension examples block back above the acknowledgments and you should be good.

File: files/en-us/mozilla/add-ons/webextensions/api/devtools/inspectedwindow/index.html [hunk 1]
```diff
@@ -46,8 +46,6 @@ <h2 id="Browser_compatibility">Browser compatibility</h2>
 
 <p>{{Compat}}</p>
 
-<p>{{WebExtExamples("h2")}}</p>
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/devtools/inspectedwindow/index.html:49]: Restore these lines, the macro is correct way of including the examples. Then delete lines 39 to 43:
```
<h2 id="Example_extensions">Example extensions</h2>
<ul>
 <li><a class="external" href="https://github.com/mdn/webextensions-examples/tree/master/devtools-panels" rel="noopener">devtools-panels</a></li>
</ul>
```

**Review by rebloor — CHANGES_REQUESTED** [review]

File: files/en-us/mozilla/add-ons/webextensions/api/devtools/inspectedwindow/index.html (hunk 1 above)
Comment (AnilSeervi) [reply files/en-us/mozilla/add-ons/webextensions/api/devtools/inspectedwindow/index.html:49]: That fixes it.

**Review by rebloor — APPROVED** [review]

**Comment by rebloor** [comment]
Thanks  @maddhruv that is looking good now

**Pull request merged**